src/pages/dashboard/globalMobile/categories/create-or-edit/index.js

```diff
@@ -145,7 +147,15 @@ export function CategoryForm() {
 
   async function handleOnFinish(values) {
     try {
-      const payload = { ...detail, ...values, field };
+      if (values.certification?.length > 0) {
+        var certificatesID = { certificates: values.certification };
+      }
```

Comment:
hmm ini yg mau di achieve apa ya?

Reply:
penggunaan `var` ga direkomendasiin karna lebih error-prone
```suggestion
      let certificatesID;
      if (values.certification?.length > 0) certificatesID = { certificates: values.certification };
```

Reply:
Ah iya bisa gitu juga, saya lupa kalau letnya bisa diatas Mba. Saya pakai var biar bisa diambil diluar scope ifnya. Makasih Mba